tools/apt-repo-status.py

```
@@ -0,0 +1,319 @@
+    import re
+    out = []
+    for m in re.findall(r'href="([^"?/]+)/"', html):
+        if m not in ("..", ".") and m not in out:
+            out.append(m)
+    return out
+
+
+def get_release(base, suite):
+    try:
+        text = fetch(f"{base}/dists/{suite}/Release").decode("utf-8", "replace")
+    except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)